app/controllers/transcribe_controller.rb

```diff
@@ -312,6 +312,37 @@ def still_editing
     render nothing: true
   end
 
+  def goto_next_untranscribed_page
+    
+    next_page_path = landing_page_path
+    flash[:notice] = "There are no more pages to transcribe in this collection! Check out some of our other projects."
+
+    if @work.next_untranscribed_page
+      flash[:notice] = "Here's another page in this work, if you'd like to continue."
+      next_page_path = collection_transcribe_page_path(@work.collection.owner, @work.collection, @work, @work.next_untranscribed_page)
+    elsif @collection.class == DocumentSet
+      docset = @collection
+      next_page = docset.find_next_untranscribed_page_for_user(current_user)
+      if !next_page.nil?
+        flash[:notice] = "There are no more pages to transcribe in this work. Here's another page in this document set if you'd like to continue."
+        next_page_path = collection_transcribe_page_path(docset.owner, docset, next_page.work, next_page)
+      else # Docset has no more Untranscribed works, bump up to collection level
+        next_page = docset.collection.find_next_untranscribed_page_for_user(current_user)
+        unless next_page.nil?
+          flash[:notice] = "There are no more pages to transcribe in this work. Here's another page in this collection if you'd like to continue."
+          next_page_path = collection_transcribe_page_path(docset.collection.owner, docset.collection, next_page.work, next_page)
+        end
+      end
+    else
+      next_page = @collection.find_next_untranscribed_page_for_user(current_user)
+      unless next_page.nil?
+        flash[:notice] = "There are no more pages to transcribe in this work. Here's another page in this collection if you'd like to continue."
```

Comment:
same comment: 
Let's get rid of "if you'd like to continue". I think ""There are no more pages to transcribe in this work. Here's another page in this collection." is fine.